### PR TITLE
Prepare five guides for Cloud users

### PR DESCRIPTION
**Pull request by ptgott**
Configuration reference

- Add a tabbed warning box, which incorporates the existing warnings
  into a single Admonition (to avoid Admonition clutter) and add
  instructions for Cloud users.

Backends

  Add a compatibility note for Cloud users. There is little else we
  can do at this point since it is not currently possible to adjust
  the visibility of an entire page of the docs site based on the scope
  selector.

EC2 node labels guide

- Add Tabs to the Prerequisites section so users don't see scope-
  irrelevant content.

- Misc. clarity/style/grammar edits.

Audit Logs guide

- Prevent Cloud, Enterprise, or OSS users from seeing scope-irrelevant
  information by using Tabs.

- Note that this change does not attempt to update the list of audit
  event types, since doing so would exceed the time I allotted for
  updating this guide.

Docker setup guide

- Remove the image.mdx partial, since it is only used once.
- Create a partial for the Enterprise Docker image table.
- Use Tabs to display different instructions for users of different
  Teleport editions.

### CODE REVIEW

File: docs/pages/setup/reference/audit.mdx [hunk 1]
```diff
@@ -3,32 +3,65 @@ title: Audit Events and Records
 description: Reference of Teleport Audit Events and Session Records
 ---
 
-Teleport logs every SSH event into its audit log. There are two components of
-the audit log:
+Teleport logs SSH, Kubernetes, and database event into its audit log. There are
```
Comment (ulysseskan) [inline docs/pages/setup/reference/audit.mdx:6]: database event**s**

**Review by ulysseskan — COMMENTED** [review]
I read through each of the changed files and everything looks good to me with the exception of one tiny grammar mistake mentioned above.  I'm ready to approve once that's fixed, if other reviewer(s) are in agreement.

**Review by ulysseskan — APPROVED** [review]
👍

File: docs/pages/setup/guides/docker.mdx [hunk 2]
```diff
@@ -1,60 +1,143 @@
...
+- Docker v(=docker.version=) or later. We currently only offer Docker images for
+  `x86_64` architectures.
 
 ```code
 $ docker version
 # Client: Docker Engine - Community
 # Version:           (=docker.version=)
 ```
 
+- The Enterprise edition of the `tsh` client tool, which ships with the `teleport` binary. Visit the [customer portal](https://dashboard.gravitational.com/web/login) to download Teleport.
```
Comment (zmb3) [inline docs/pages/setup/guides/docker.mdx:46]: There is no such thing as the enterprise edition of `tsh`. Only `tctl` and `teleport` have enterprise editions.

File: docs/pages/setup/guides/ec2-tags.mdx [hunk 3]
```diff
@@ -1,20 +1,26 @@
 ---
-title: EC2 Tags as Teleport node labels
-description: How to setup Teleport node labels based on EC2 tags
-h1: Sync EC2 Tags and Teleport node labels
+title: EC2 Tags as Teleport Node Labels
```
Comment (zmb3) [inline docs/pages/setup/guides/ec2-tags.mdx:2]: Just a note - we'll be adding a first-class feature to Teleport that does this in Teleport 10 or 11, at which point this guide will become deprecated.

File: docs/pages/setup/reference/audit.mdx [hunk 4]
```diff
@@ -3,32 +3,65 @@ title: Audit Events and Records
...
 
-1. **Cluster Events:** Teleport logs events like successful user logins along with the metadata like remote IP address, time, and the session ID.
-2. **Recorded Sessions:** Every SSH or Kubernetes shell session is recorded and can be replayed later. The recording is done by the nodes themselves, by default,
+<Tabs>
+<TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
+
+
+1. **Cluster Events:** Teleport logs events like successful user logins along
+   with metadata like remote IP address, time, and the session ID.
+2. **Recorded Sessions:** Every SSH or Kubernetes shell session is recorded and
```
Comment (zmb3) [inline docs/pages/setup/reference/audit.mdx:15]: This also applies to desktop sessions.

**Review by zmb3 — CHANGES_REQUESTED** [review]

File: docs/pages/setup/guides/ec2-tags.mdx (hunk 3 above)
Comment (ptgott) [reply docs/pages/setup/guides/ec2-tags.mdx:2]: Created an issue to remove this when needed: https://github.com/gravitational/teleport/issues/11929

**Review by zmb3 — APPROVED** [review]

**Pull request merged**